spec/pages/index.spec.js

```diff
@@ -0,0 +1,32 @@
+import { shallowMount } from '@vue/test-utils';
+import PageIndex from '~/pages/index';
+import { sampleMembers } from '~/spec/sample-data/members';
+
+describe('IndexPage', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = shallowMount(PageIndex, {
+      data() {
+        return {
+          members: sampleMembers,
+        };
+      },
+    });
```

Comment:
Instead of passing data directly here, do you want to try mocking the fetch response?